packages/app/src/providers/mock/mock-api.ts

```diff
@@ -50,16 +50,7 @@ export class MockApi extends DataProviderApi {
     return entity;
   }
 
-  public async getAttrValues(entity: Entity): Promise<AttributeValues> {
```

Comment:
Just moving this down below `getValue` for consistency between all provider API classes.